internal/ask/form/submission/submission.go

```diff
@@ -138,6 +140,41 @@ func (s *Submission) Validate() error {
 	return nil
 }
 
+// GetQuestions retrieves the questions
+func (s *Submission) GetQuestions() []string {
+	var h []string
+
+	for _, r := range s.Answers {
+		h = append(h, r.Question.(string))
+	}
+
+	return h
+}
+
```

Comment:
Yes. You are right! I'm going to change it to a function and merge it into only one function to address your second comment.